src/core/src/package_managers/YumPackageManager.py

```diff
@@ -940,21 +940,5 @@ def do_processes_require_restart(self):
         return process_count != 0  # True if there were any
     # endregion Reboot Management
 
-    def add_arch_dependencies(self, package_manager, package, packages, package_versions, package_and_dependencies, package_and_dependency_versions):
```

Comment:
This method got removed while merging from latest code. Similarly in ZypperPackageManager.py

Reply:
Added back. Thanks